.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Is this comment block the "Notice" required for redistribution?

Reply:
Yes, this file is generated by the [Maven Wrapper](https://github.com/takari/maven-wrapper) and is intended to be checked into vcs